corehq/ex-submodules/couchforms/models.py

```diff
@@ -137,6 +137,7 @@ class XFormInstance(DeferredBlobMixin, SafeSaveDocument, UnicodeMixIn,
     date_header = DefaultProperty()
     build_id = StringProperty()
     export_tag = DefaultProperty(name='#export_tag')
+    _blobdb_type_code = CODES.form_xml
```

Comment:
forms an have `form_xml` and `form_attachment` right? How is that handled?

Reply:
Good catch. eb85c0992c54ece89c70879ab7c83657e8d91e0e